uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -198,6 +204,57 @@ define([
       });
     });
 
+    describe('$scope.onContactChanged', function () {
+      beforeEach(function () {
+        spyOn(ContactService, 'updateCache').and.returnValue({});
+        spyOn(AssignmentService, 'search').and.returnValue($q.resolve([]));
+      });
+
+      beforeEach(function () {
```

Comment:
You can keep a single `beforeEach`and give a line break to define different sections.